canonical/snapcraft.io
# Add expandable nav to docs

## Done
- Removed unused files
- Deduped some of the code
- Implemented expandable navigation

## How to QA
- Visit https://snapcraft-io-4192.demos.haus/docs and ensure everything works as expected.
- Also ensure the search results page looks OK
- Make sure things are as expected on mobile

## Issue / Card
Fixes #4184 
and https://warthogs.atlassian.net/browse/WD-1135

## Screenshots
